spec/requests/feedbacks_spec.rb

```diff
@@ -211,12 +211,18 @@
 
       describe "when three feedbacks are submitted in quick succession" do
         it "returns a 429 status for the final submission" do
+          freeze_time
+
           post feedback_index_path, params:, headers: { "HTTP_REFERER" => originating_page }
           expect(response).to have_http_status(:ok)
           expect(Feedback.count).to eq 1
+
           post feedback_index_path, params:, headers: { "HTTP_REFERER" => originating_page }
           expect(response).to have_http_status(:ok)
           expect(Feedback.count).to eq 2
+
+          travel_to 11.seconds.from_now
```

Comment:
I don't like magic numbers, but I didn't think it was worth introducing a constant to the rack-attack initialiser for this test alone.

If you do, feel free to add it and merge this in my absence.